### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by vivbak**
* Update pip version to pip3.10
* Fix interactive install for python3.10
* Update python3-dev to python3.10 dev.

### CODE REVIEW

**Comment by vladsaveliev** [comment]
Can we actually update `driver-base` to use python 3.10?

File: images/cpg-pipes/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
     build-essential \
-    python3 \
-    python3-pip \
-    python3-dev \
+    curl \
     && \
-    pip install --no-cache-dir cpg-pipes==0.3.1 && \
+    add-apt-repository ppa:deadsnakes/ppa -y && \
+    apt-get install -y --no-install-recommends python3.10 python3.10-distutils python3-apt python3.10-dev  && \
+    curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10 && \
```
Comment (vladsaveliev) [inline images/cpg-pipes/Dockerfile:15]: Can't we install pip for python 3.10 with apt-get?

Comment (vladsaveliev) [reply images/cpg-pipes/Dockerfile:15]: E.g. even if `python3.10` is installed, `python3-pip` would still somehow use python 3.8?

File: images/cpg-pipes/Dockerfile [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+    pip3.10 install --no-cache-dir cpg-pipes==0.3.1 && \
```
Comment (vladsaveliev) [inline images/cpg-pipes/Dockerfile:16]: Wondering if it's possible to make `pip` and `python` point to `pip3.10` and `python3.10` by default, so we don't need to worry about python versions when running python scripts

**Comment by vladsaveliev** [comment]
Another option would be to change the base to `FROM python:3.10`

**Review by vladsaveliev — APPROVED** [review]

File: images/cpg-pipes/Dockerfile (hunk 2 above)
Comment (vivbak) [reply images/cpg-pipes/Dockerfile:16]: I had a really difficult time doing this, but very open to suggestions.

**Pull request merged**